{
  Class<? extends Throwable> exception=YarnTestException.class;
  verifyRemoteExceptionUnwrapping(exception,exception.getName());
}
